src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../essentials';
+import { Box } from '../Box/Box';
+import Compact from './components/Compact';
+import DefaultPanel from './components/Default';
+
+interface AccordionProps {
+    heading?: string;
```

Comment:
Are these types correct? Should we be limiting to a string or is it acceptable to pass through a react element here as well? Perhaps React.ReactNode fits better.

Reply:
The `heading` is just the header title. I'll renamed it to avoid confusions

Reply:
Yeah but it doesn't make sense to me to restrict it, for example, if you wanted to include an icon within the heading or something along those types of lines would that be an issue?

Reply:
In the design the title only can be a text, that is why that case is not contemplated

Reply:
In the syncs with design we came to the conclusion that this component would be restricted in multiple aspects, e.g. title and label not accepting nodes, just strings. I suppose if in the future there appear new needs we would extend the component

Reply:
I'm a little confused as to why we'd restrict this now? I mean if a designer presents a design where we need something other than a string it creates more work later. Whereas having nodes now doesn't stop a string being entered. Also any use of this component will be checked by a designer.

Reply:
This was restricted by the designers, not by us. As far as i remember it was their initiative to follow this same pattern (with just a string) and commit to only using it in this manner, the idea in the early conversations about the Accordion was to accept a node as the header but it was changed. Of course in case a designer needs something other than a string then we'll need to change it but in principle it shouldn't happen. Please correct me @rafael-sepeda

Reply:
Sorry @martimalek for late reaction.

@christopherrolfe198 yes, for now we want to have want to have a limited set of variants. Agree that in general we might need more flexibility in the future. However to make the setup more easy to implement we decided to compromise the flexibility.

By design we have a limited list of variants
https://www.figma.com/file/dTefUItBx5bfbxcCcMzMXm/Wave-DS?node-id=1973%3A67543

Reply:
I think that because @christopherrolfe198 is on vacation and design has approved it we can mark this as resolved